src/components/contact-govuk-one-login/contact-govuk-one-login-controller.ts

```diff
@@ -35,6 +36,7 @@ export function contactGet(req: Request, res: Response): void {
     showSignOut: isAuthenticated && !isLoggedOut,
     referenceCode,
     contactEmailServiceUrl: contactEmailServiceUrl,
+    webchatSource: getWebchatUrl(),
```

Comment:
does contactWebchatEnabled need adding to the data?

Reply:
It's already there, but it's not shown in this short diff because we have so many variables in the data for this page